examples/step-60/step-60.cc

```diff
@@ -0,0 +1,1077 @@
+// class. The structure of deal.II, as many modern numerical libraries, is
+// organized following a Directed Acyclic Graph (DAG). A DAG is a directed graph
+// with topological ordering: each node structurally represents an object, and
+// one or more directed edges represent how it can be used to generate new
+// objects. This has several advantages, but intrinsically creates “asymmetries”
+// as certain operations are fast unlike their inverse. For example, in deal.II
+// finding the vertices of a cell has very low computational cost, while finding
+// all the cells that share a vertex requires a non-trivial computation unless a
+// new data structure is added.
+//
+// Since inverse operations are usually not needed in a finite element code,
+// these are implemented in GridTools without the use of extra data structures
+// related to the Triangulation which would make them much faster. One such data
+// structure, for example, is a map from the vertices of a Triangulation to all
+// cells that share those vertices, which would reduce the computations needed
+// to answer to the previous example.
+//
+// Some methods, for example GridTools::find_active_cell_around_point, make
+// heavy usage of these non-standard operations. If you need to call these
+// methods more than once, it becomes convenient to store those data structures
+// somewhere. GridTools::Cache does exactly this, giving you access to
+// previously computed objects, or computing them on the fly (and then storing
+// them inside the class for later use), and making sure that whenever the
+// Triangulation is updated, also the relevant data strucutres are recomputed.
+
+#include <deal.II/fe/fe.h>
+#include <deal.II/fe/fe_q.h>
+#include <deal.II/fe/fe_system.h>
+
+#include <deal.II/fe/mapping_q_eulerian.h>
+#include <deal.II/fe/mapping_fe_field.h>
+
+// In this example, we will be using a reference domain to describe an embedded
+// Triangulation, deformed through a finite element vector field.
+//
+// The two include files above contain the definition of two classes that can be
+// used in these cases. MappingQEulerian allows one to describe a domain through
+// a *displacement* field, based on a FESystem[FE_Q(p)^spacedim] finite element
+// space. The second is a little more generic, and allows you to use arbitrary
+// vector FiniteElement spaces, as long as they provide a *continuous*
+// description of your domain. In this case, the description is done through the
+// actual *deformation* field, rather than a *displacement* field.
+//
+// Which one is used depends on how the user wants to specify the reference
+// domain, and/or the actual configuration. We'll provide both options, and
+// experiment a little in the results section of this tutorial program.
+
+#include <deal.II/dofs/dof_tools.h>
+
+#include <deal.II/base/parsed_function.h>
+
+// The parsed function class is another new entry. It allows one to create a
```

Comment:
same issue here -- make it clear that this comment refers to the previous include file.